<html>
 <head></head>
 <body>
  <p>We now make a final attempt to find <em>DnaA</em> boxes in <em>E. coli</em> by finding the most frequent 9-mers with mismatches and reverse complements in the region suggested by the minimum skew as <em>ori</em>. Although the minimum of the skew diagram for <em>E. coli</em> is found at position 3923620, we should not assume that its <em>ori</em> is found exactly at this position due to random fluctuations in the skew. To remedy this issue, we could choose a larger window size (e.g., <em>L</em> = 1000), but expanding the window introduces the risk that we may bring in other clumped 9-mers that do not represent <em>DnaA</em> boxes but appear in this window more often than the true <em>DnaA</em> box. It makes more sense to try a small window either starting, ending, or centered at the position of minimum skew.</p> 
  <p>Letâ€™s cross our fingers and identify the most frequent 9-mers (with 1 mismatch and reverse complements) within a window of length 500 starting at position 3923620 of the <a href="http://bioinformaticsalgorithms.com/data/realdatasets/Replication/E_coli.txt" rel="nofollow noopener noreferrer" target="_blank"><em>E. coli</em> genome</a>. Bingo! The experimentally confirmed <em>DnaA</em> box in <em>E. coli</em> (<samp><span style="color:olivedrab"><strong>TTATCCACA</strong></span></samp>) is indeed a most frequent 9-mer with 1 mismatch, along with its reverse complement <samp><span style="color:saddlebrown"><strong>TGTGGATAA</strong></span></samp>:</p> 
  <p style="text-align: center;"><samp>aatgatgatgacgtcaaaaggatccggataaaacat<strong>ggtgattgcctcgc</strong><br><strong>ataacgcggtatgaaaatggattgaagcccgggccgtggattctactcaa</strong><br><strong>ctttgtcggcttgagaaagacctgggatcctgggtattaaaaagaagatc</strong><br><strong>tatttatttagagatctgttctattgtgatctcttattaggatcgcactg</strong><br><strong>ccc<span style="color:saddlebrown">TGTGGATAA</span>caaggatccggcttttaagatcaacaacctggaaagga</strong><br><strong>tcattaactgtgaatgatcggtgatcctggaccgtataagctgggatcag</strong><br><strong>aatgagggg<span style="color:olivedrab">TTAT<font color="red">A</font>CACA</span>actcaaaaactgaacaacagttgttc<span style="color:saddlebrown">T<font color="red">T</font>TGGA<br> TAA</span>ctaccggttgatccaagcttcctgacagag<span style="color:olivedrab">TTATCCACA</span>gtagatcg</strong><br> cacgatctgtatacttatttgagtaaattaacccacgatcccagccattc<br> ttctgccggatcttccggaatgtcgtgatcaagaatgttgatcttcagtg</samp></p> 
  <p>You will notice that we emphasized a large part of this sequence with bold text. This region is the experimentally verified <em>ori</em> of <em>E. coli</em>, which starts 37 nucleotides after position 3923620, where the skew reaches its minimum value.</p> 
  <p>We were very fortunate that the <em>DnaA</em> boxes of <em>E. coli</em> are captured in the window that we chose. Moreover, while <samp><span style="color:olivedrab"><strong>TTATCCACA</strong></span></samp> represents a most frequent 9-mer with 1 mismatch and reverse complements in this 500-nucleotide window, it is not the only one: <samp><strong>GGATCCTGG</strong></samp>, <samp><strong>GATCCCAGC</strong></samp>, <samp><strong>GTTATCCAC</strong></samp>, <samp><strong>AGCTGGGAT</strong></samp>, and <samp><strong>CTGGGATCA</strong></samp> also appear 4 times with 1 mismatch and reverse complements.</p> 
  <p><strong><span style="color: #ff4363;">STOP and Think:</span></strong> In this chapter, every time we find <em>ori</em>, we seem to find some other surprisingly frequent 9-mers. Why do you think this is?</p>
 </body>
</html>